bindata/etcd/etcd-common-tools

```diff
@@ -42,7 +42,7 @@ function check_snapshot_status() {
   local snap_file="$1"
   if ! etcdctl snapshot status "${snap_file}" -w json; then
     echo "Backup integrity verification failed. Backup appears corrupted. Aborting!"
-    return 1
+    exit 1
```

Comment:
A more proper fix would be to check where this is called and exit with a non-zero status there in case of an error

Reply:
+1